docs/contrib/CI.md

```diff
@@ -0,0 +1,81 @@
+# CI Guide
+
+This document describes the pipelines available to the developers and
+code maintainers. Note that pipelines with the suffix `*-Private` are only
+available to code maintainers and The OpenROAD Project members. To access
```

Comment:
I would mention why they are private (PDK access) so users see the need for the restriction and don't feel excluded.

Reply:
Explanation added on [d702c8d](https://github.com/The-OpenROAD-Project/OpenROAD/pull/1740/commits/d702c8dabcbcc16200a3be9db2b96b98ec49c7b9)